### ISSUE TITLE
Offer tutorial modal for first timers

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#5403]
- We should offer first timer direct links to tutorials 

It could be done as notification/toast message

- we would need copy for text in toast, maybe @JustLuuuu can drop bites  
- Consider adding link Tutorials in Navbar 
- Style could be resued like is in congratulations on first mint and first buy
- Check if first timer has minted any nfts, if none, show this tutorial toast/modal

**Comment by VladoTheBoi** [comment]
Do we have a testnet of KodaDot on some of the testnet chains? 
I though that it might be cool to incentivize people to try and play around with the marketplace before they engage with their own money. Also the process of topping up money into their wallet might be one of the problems they dont engage at all. 

1. this could be done with a testnet (also could have more fictional money to also buy some NFTs etc.) 
2. or I though that we might offer first timers and people who hasnt minted anything some small amount of KSM (enough to create a collection and mint 1 NFT into it) so that they could atleast try to mint something. 

This might then incentivize them more with engaging with KodaDot. We would need to think of some restrictions in case of the 2. option so that it wouldnt be targeted and farmed with bots etc. We could also come then come up with some nice and easy on-site tutorial which could help then along the way.

Just a thought :)

**Comment by exezbcz** [comment]
yy, that is similar to this issue 
- #4877

**Comment by VladoTheBoi** [comment]
Yep thats basically it, not sure whether 20 mints isnt too much but this could be reduced if needed.
